layers/+email/mu4e/packages.el

```diff
@@ -50,7 +49,8 @@
       (setq mu4e-completing-read-function 'completing-read
             mu4e-use-fancy-chars 't
             mu4e-view-show-images 't
-            message-kill-buffer-on-exit 't)
+            message-kill-buffer-on-exit 't
+            mu4e-org-support nil)
```

Comment:
Can you tell why you are disabling `mu4e-org-support` by default here?

Reply:
Setting this variable causes `(require 'mu4e-org)` (see [src](https://github.com/djcb/mu/blob/master/mu4e/mu4e.el#L40)) which causes `(require 'org)`.

One may not want to load org right when mu4e is loaded. In fact the only reason I can think of for triggering loading org from mu4e is when you want to write in org-mode syntax in the message buffer, which is handled by the `org-mu4e` feature, loaded explicitly via `org-mu4e-compose-org-mode`.

Reply:
On the other hand, because `mu4e-org` merely enables capturing mu4e messages and views as org links, it could simply be loaded with org (`:after org`).

Reply:
Ok, this sounds right, lets keep this deactivated otherwise we break lazy loading of org.